streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -240,14 +241,12 @@ public class StreamsConfig extends AbstractConfig {
     public static final String CLIENT_TAG_PREFIX = "client.tag.";
 
     /** {@code topology.optimization} */
-    private static final String CONFIG_ERROR_MSG = "Acceptable values are:"
```

Comment:
Just moved below -- the above `JavaDoc` belongs to `TOPOLOGY_OPTIMIZATION_CONFIG` and we should fix this in any case...